pkgs/development/python-modules/euclid3/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+}:
+buildPythonPackage rec {
+  pname = "euclid3";
+  version = "0.01";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-JbgnpXrb/Zo/qGJeQ6vD6Qf2HeYiND5+U4SC75tG/Qs=";
+  };
+
+  pythonImportsCheck = [
+    "euclid3"
+  ];
+
+  meta = with lib; {
+    description = "2D and 3D vector, matrix, quaternion and geometry module.";
+    homepage = "http://code.google.com/p/pyeuclid/";
```

Comment:
that site returns a 500

Reply:
Maybe temporary outage? Here is what I see:
```
> curl -iL http://code.google.com/p/pyeuclid/
HTTP/1.1 301 Moved Permanently
Content-Type: text/html; charset=utf-8
Location: https://code.google.com/archive/p/pyeuclid
X-Cloud-Trace-Context: 01e9eeda7ec89b111113a686f0e3a287
Date: Mon, 28 Aug 2023 06:22:42 GMT
Server: Google Frontend
Content-Length: 291

HTTP/2 200 
date: Mon, 28 Aug 2023 06:22:42 GMT
expires: Mon, 28 Aug 2023 06:32:42 GMT
cache-control: public, max-age=600
etag: "eunJCg"
x-cloud-trace-context: 798e5f2709bfceacaa407394ef4bef2b
content-type: text/html
server: Google Frontend
alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
```

Reply:
Marking this resolved. Please reopen if this is still giving you a 500 somehow.